components/user/UserPage.js

```diff
@@ -6,13 +6,15 @@ import UserLinks from "./UserLinks";
 import UserMilestones from "./UserMilestones";
 import UserTestimonials from "./UserTestimonials";
 import UserEvents from "./UserEvents";
+import UserEddiehubData from "./UserEddiehubData";
 
-export default function UserPage({ data, BASE_URL }) {
+export default function UserPage({ data, BASE_URL, apiEvents }) {
   const defaultTabs = [
     { name: "My Links", href: "#", current: true },
     { name: "Milestones", href: "#", current: false },
     { name: "Testimonials", href: "#", current: false },
     { name: "Events", href: "#", current: false },
+    { name: "Eddiehub Stats", href: "#", current: false },
```

Comment:
I'm totally open to naming here, feel free to give other suggestions if this doesnt work

Reply:
Most interesting to follow 🥳
Good work, @Cahllagerfeld 🥳🤩🥰

Reply:
I like it, but the `h` should be capital `EddieHub`